PythonRegression/tests/features/steps/machine_configuration_steps.py

```diff
@@ -24,4 +21,10 @@ def configuration():
         machine = nodes
           
     world.machine = machine
-    
+    world.config = {}
+    world.responses = {}
+
+@after.each_example
+def deconfiguration(scenario,outline,steps):
+    world.config = {}
+    world.responses = {}
```

Comment:
This will work.
However, what will happen if you have other fields in world you want to clear in the future? you will have to add to this code.
Maybe consider removing everything in the dictionary besides `machine`?